server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,76 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+To use xref:_oidc[OpenID Connect], an existing application must be secured as an Open ID connect client.
+
+.Procedure
+. Click `Clients` in the left navigation pane.  
++
+.Clients
+image:{project_images}/clients.png[Clients]
+
+. Click *Create* to go to the `Add Client` page.
++
+.Add Client
+image:{project_images}/add-client-oidc.png[Add Client]
+
+. Enter the `Client ID` of the client.
+
+. Select `openid-connect` in the `Client Protocol` drop down box.
+
+. Enter the base URL of your application in the `Root URL` field.
+
+. Click *Save*.  This creates the client and bring you to the `Settings`
+tab.
++
+.Settings
+image:{project_images}/client-settings-oidc.png[Client Settings]
++
+`Client ID`:: The alpha-numeric ID string that is used in OIDC requests and in the {project_name} database to identify the client.
+
+`Name`:: The name for the client in {project_name} UI screen. To localize
+the name, set up a replacement string value. For example, $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}] for more information.
+
+`Description`:: The description of the client.  This can also be localized.
+
+`Enabled`:: When turned off, the client cannot request authentication.
+
+`Consent Required`:: When turned on, users see a consent page that they can use to grant access to that application.  It will also display metadata so that the user knows exactly what information the client is getting access to.
```

Comment:
"... exactly what information the client is getting access to."  ends with a preposition.  I suggest "the exact information that the client can access."